src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.spec.ts

```diff
@@ -70,7 +70,13 @@ describe('RgwBucketFormComponent', () => {
     }));
 
     it('bucket names cannot be formatted as IP address', fakeAsync(() => {
-      const testIPs = ['1.1.1.01', '001.1.1.01', '127.0.0.1'];
+      const testIPs = [
+        '1.1.1.01',
+        '001.1.1.01',
+        '127.0.0.1',
+        '2001:db8:3333:4444:5555:6666:7777:8888',
+        '2001:db8:3333:4444:CCCC:DDDD:EEEE:FFFF'
```

Comment:
I don't get why we need to validate against IPv6 addresses when `:` is not a supported char. So the charset rule is enough to filter out IPv6 addresses.

Reply:
You are right. We don't need to check that. I'll remove it then.